specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -44,6 +45,28 @@ export class BusinessRuleManager<SCHEMA extends AnySchema> {
     }
   }
 
+  private added(
+    resource: SpecifyResource<SCHEMA>,
+    collection: Collection<SCHEMA>
+  ) {
+    if (resource.specifyModel.getField('ordinal') !== undefined)
```

Comment:
this is a bit of a code smell. can you double check that?
- the code on the back-end does not make this code on the front-end redundant
- there is no simple way to accomplish this via the business rules definitions .json file that you created so that this is defined once and used both by back-end and front-end

Reply:
> the code on the back-end does not make this code on the front-end redundant
Yes, the only reason I added the code to the frontend was because that's what was causing the original issue. 

I figured also having the code on the frontend would alleviate some other potential bugs because the frontend changes are more general. 
I can remove either the frontend or backend changes to cut down on redundancy

Reply:
as you see best
you know the business rules code better than me

Reply:
That would also allow me to remove a lot of redundant code on the backend... 
There are multiple cases where we set the ordinal/ordernumber property if it doesn't exist. Might be nice to remove them